File: CSharp/SpaceVIL/SpaceVIL/Cores/PixMapData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpaceVIL
{
    class PixMapData
    {
        private List<float> pixels;
        private List<float> colors;
        private float alpha;

        public PixMapData(List<float> pixels, List<float> colors, float alpha)
        {
            this.pixels = pixels;
            this.colors = colors;
            this.alpha = alpha;
        }

        public List<float> GetPixels()
        {
            return pixels;
        }

        public List<float> GetColors()
        {
            return colors;
        }

        public float GetAlpha()
        {
            return alpha;
        }
    }
}

File: Java/Fonts/PixMapData.java
package CommonVesselGUI.Cores;

import java.util.List;

public class PixMapData {
    private List<Float> pixels;
    private List<Float> colors;
    private float alpha;

    public PixMapData(List<Float> pixels, List<Float> colors, float alpha) {
        this.pixels = pixels;
        this.colors = colors;
        this.alpha = alpha;
    }

    public List<Float> getPixels() {
        return pixels;
    }

    public List<Float> getColors() {
        return colors;
    }

    public float getAlpha() {
        return alpha;
    }
}
